### PR TITLE
eri9600/buffer sample

### PR DESCRIPTION
**Pull request by eri9600**
Buffer sample as shown here https://devtopia.esri.com/runtime/common-samples/issues/195

### CODE REVIEW

File: java/buffer/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+                android:id="@+id/mapView"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:paddingTop="40dp">
+        </com.esri.arcgisruntime.mapping.view.MapView>
+        <EditText
+                android:id="@+id/buffer_input"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:hint="Specify the buffer in miles then tap on map "
```
Comment (TADraeseke) [inline java/buffer/src/main/res/layout/activity_main.xml]: Extract to string resource

File: java/buffer/src/main/java/com/esri/arcgisruntime/sample/buffer/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    // create a buffer around the clicked location
+    mMapView.setOnTouchListener(new DefaultMapViewOnTouchListener(this,mMapView){
+      @Override
+      public boolean onSingleTapConfirmed(MotionEvent motionEvent){
+        // get the point that was clicked and convert it to a point in the map
+        android.graphics.Point clickLocation = new android.graphics.Point(Math.round(motionEvent.getX()),
+            Math.round(motionEvent.getY()));
+        Point mapPoint = mMapView.screenToLocation(clickLocation);
+        // only draw a buffer if a value was entered
+        if(mBufferInput.getText().toString().length() !=0){
```
Comment (TADraeseke) [inline java/buffer/src/main/java/com/esri/arcgisruntime/sample/buffer/MainActivity.java]: replace with '!mBufferInput.getText().toString().isEmpty()' for clarity

**Comment by eri9600** [comment]
@TADraeseke I fixed the items you pointed out. Thanks!

File: java/buffer/src/main/java/com/esri/arcgisruntime/sample/buffer/MainActivity.java [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+          Polygon bufferGeometry = GeometryEngine
+              .buffer(mapPoint,miles.convertTo(meters,Double.valueOf(mBufferInput.getText().toString())));
+          // show the buffered region as a green graphic
+          Graphic bufferGraphic = new Graphic(bufferGeometry,fillSymbol);
+          graphicsOverlay.getGraphics().add(bufferGraphic);
+          // show a red marker where clicked
+          Graphic markerGraphic = new Graphic(mapPoint, new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.CIRCLE,
+              0xFFFF0000, 5));
+          graphicsOverlay.getGraphics().add(markerGraphic);
+        }
```
Comment (TADraeseke) [inline java/buffer/src/main/java/com/esri/arcgisruntime/sample/buffer/MainActivity.java]: add an else. Something  like:

```
} else {
          Toast.makeText(MainActivity.this, "Please enter a buffer distance first.", Toast.LENGTH_LONG).show();
        }
```

File: java/buffer/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+        </com.esri.arcgisruntime.mapping.view.MapView>
+        <EditText
+                android:id="@+id/buffer_input"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:hint="@string/Enter_miles_string"
+                android:inputType="number"
+                android:layout_alignTop="@id/mapView"/>
+
+
```
Comment (TADraeseke) [inline java/buffer/src/main/res/layout/activity_main.xml]: can remove this whitespace

**Comment by eri9600** [comment]
@TADraeseke Fixed the changes suggested. Going to merge with master.

**Pull request merged**